masonry/src/passes/event.rs

```diff
@@ -51,6 +52,7 @@ fn run_event_pass<E>(
             widget_state: state_mut.item,
             widget_state_children: state_mut.children,
             widget_children: widget_mut.children,
+            target: original_target.unwrap(),
```

Comment:
Why is the `unwrap` here rather than at the definition of `original_target`?

Reply:
Because a value of None is valid at that point.

If `original_target` is None, then the loop is never entered.

Reply:
Thanks. Sorry for the confusion